CooperationHullMainSite/Views/Home/Membership.cshtml

```diff
@@ -5,10 +5,91 @@
 }
 
 <div class="row">
-    <div class="col-12 content-text content-page">
-        <h1 class="heading-xl">@ViewData["Title"]</h1>
+    <article class="col-12 content-page px-md-5">
+        <div class="row pt-5">
+            <h1 class="heading-xl">@ViewData["Title"]</h1>
+            <p>If you like what you’ve heard, if you’re just as worried and hopeful about the world as we are, and want to support our work, join us!</p>
+        </div>
 
-        <p>Coming Soon</p>
-    </div>
+        <div class="pt-5 d-flex flex-column gap-8">
+            <section>
+                <div class="row">
+                    <h2 class="pb-4">Anyone can become a member of Cooperation Hull</h2>
+                </div>
+                <div class="row gap-4 gap-lg-0">
+                    <div class="col-12 col-lg-6">
+                        <p>You’ll join a ramshackle community of strangers, from builders to poets, who all have one thing in common: we want to build a better world and we’re ready to roll up our sleeves.</p>
+                        <p>Your monthly contributions will keep us going through thick and thin. Regular income, no matter how small, helps us plan into the future. And the knowledge that we have friends supporting us will help motivate us to keep knocking on doors, keep asking the hard questions, keep listening, keep trying and keep experimenting.</p>
+                        <a class="button-link" asp-area="" asp-controller="Home" asp-action="Index">Read more about what we're doing &rarr;</a>
```

Comment:
Interestingly the element's text here is shorter in height, but fine when we render the same `a` element in the contribution boxes below 🤔